app/views/orga/teams/show.html.slim

```diff
@@ -74,12 +74,11 @@ nav.btn-group
   h3 Supervisors
   p = @team.supervisors.any? ? link_to_team_members(@team, :supervisor) : content_tag(:em, 'Will be added by RGSoC Organizers')
 
-- if @team.members.any? { |user| user.conferences.in_current_season.any? }
+- if @team.conferences.in_current_season.any?
   h4 Conferences
   ul.conferences
-    - @team.members.each do |user|
-      - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(', ')}".html_safe
+      - if @team.conferences.in_current_season.any?
```

Comment:
Maybe I don't get it, but I think you can spare this line, since you already have this condition for the whole block 3 lines above.